cmd/aries-js-worker/main.go

```diff
@@ -104,7 +103,8 @@ func takeFrom(in chan *command) func(js.Value, []js.Value) interface{} {
 
 			return nil
 		}
-		in <- cmd
+
+		go func() { in <- cmd }()
```

Comment:
@troyronda @llorllale  I did another fix. It prevents deadlock.

Reply:
This seems to make more sense. See https://github.com/golang/go/issues/34478

Reply:
@soluchok how many times did you try it? What was the failure rate?

Reply:
Also - note the channel is unbuffered.

Reply:
@llorllale In progress. It seems to work fine. For now 7/7. I am gonna run it 100 times.